README.md

```
@@ -175,6 +210,36 @@ Example:
 
 ![](https://i.imgur.com/DfA2lcd.gif)
 
+## Example Gallery
+
+### Terminal Background Samples
+
+Pikachu:
+
+![](https://i.imgur.com/n34fXyp.png)
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)